packages/ui/src/components/IconButton/iconButton.tsx

```diff
@@ -0,0 +1,69 @@
+import {
+    Button as DefaultButton,
+    ButtonProps as DefaultButtonProps,
+} from 'antd'
+import classNames from 'classnames'
+import React, { useCallback } from 'react'
+
+import { sendAnalyticsClickEvent, extractChildrenContent } from '../_utils/analytics'
+import { BUTTON_CLASS_PREFIX } from '../Button/button'
+
+const ICON_BUTTON_CLASS_PREFIX = 'condo-icon-btn'
+
+type CondoIconButtonProps = {
+    focus?: boolean
+    size?: 'small' | 'medium'
+    pressed?: boolean
+}
+
+export type IconButtonProps = Omit<DefaultButtonProps, 'shape' | 'size' | 'style' | 'ghost' | 'type' | 'prefix' | 'prefixCls' | 'icon' | 'danger' | 'block'>
+& CondoIconButtonProps
+
+const IconButton: React.ForwardRefExoticComponent<IconButtonProps & React.RefAttributes<HTMLButtonElement>> = React.forwardRef((props, ref) => {
+    const { className, children, onClick, id, focus, size, pressed, ...rest } = props
+    const mergedSize = size || 'medium'
+    const ariaLabel = rest['aria-label']
+    const classes = classNames(
+        {
+            [BUTTON_CLASS_PREFIX]: true,
+            [`${ICON_BUTTON_CLASS_PREFIX}-pressed`]: pressed,
+            [`${ICON_BUTTON_CLASS_PREFIX}-focus`]: focus,
+            [`${ICON_BUTTON_CLASS_PREFIX}-${mergedSize}`]: mergedSize,
+        },
+        className,
+    )
+
+    const wrappedContent = children
+        ? <span className={`${ICON_BUTTON_CLASS_PREFIX}-content`}>{children}</span>
+        : null
```

Comment:
I think it's better to pass icon from prop (like `icon={PlusCircle}`, `size` of icon = `size` of IconButton) and disable children prop. Now you can pass any node in children

Reply:
You can also pass any node to `icon` prop.

Your icon can be custom: svg, font icon, icon from other packages, because `Icon` size and `Button` size it's two different params and not depends each other